.gitignore

```diff
@@ -28,6 +28,8 @@ pubspec.lock
 docs/doxygen/
 xcuserdata
 
+# directories pulled in via deps or hooks
+third_party/glfw/
```

Comment:
Is this needed after https://github.com/flutter/engine/pull/46729 ?

Reply:
Nope -- good catch!

Reply:
Nope, we fixed it!